quickstart/README.md

```diff
@@ -0,0 +1,39 @@
+# Quick Start App
+
+The quick start application is a **self-contained and runnable** example app that demonstrates authenticating, connecting to, administering, ingesting data into and querying Azure Data Explorer using the azure-kusto-python SDK.
+You can use it as a baseline to write your own first kusto client application, altering the code as you go, or copy code sections out of it into your app.
+
+**Tip:** The app includes comments with tips on recommendations, coding best practices, links to reference materials and recommended TODO changes when adapting the code to your needs.
+
+
+## Using the App for the first time
+
+### Prerequisites
+1. Set up Java on your machine. For instructions on how to install Java, consult a Java environment setup tutorial, like [this one](https://www.tutorialspoint.com/java/java_environment_setup.htm).
+2. Set up [Apache Maven](https://maven.apache.org/install.html), which is the most popular Java dependency management tool.
+
+### Retrieving the app from GitHub
+1. Download the app files from this GitHub repo
+2. Modify `kusto-sample-config.json`, changing `KustoUri`, `IngestUri` and `DatabaseName` appropriately for your ADX cluster

```

Comment:
I had named the instruction `ONECLICK_INSTRUCTION_BOX.md` in line with the md file convention, but you asked me to rename for consistency with Python.

So the question here is similar - the filename `kusto-sample-config.json` uses hyphens instead of underscores, per Java's convention. However, the file in Python is slightly different in line with Python's conventions - `kusto_sample_config.json`.

Change this to be identical to the Python file's name?

Reply:
Tal requested that the filenames be identical.